### PR TITLE
about-page: add links for zulip-electron and license

### PR DESCRIPTION
**Pull request by priyank-p**
screen shot 2017-12-02 at 2 04 37 pm
Refs: https://github.com/zulip/zulip-electron/issues/244

### CODE REVIEW

File: app/renderer/css/about.css [hunk 1]
```diff
@@ -44,15 +46,13 @@ body {
 .maintenance-info {
 	position: absolute;
 	width: 100%;
-	bottom: 20px;
 	left: 0px;
 	color: #444;
 }
```
Comment (akashnimare) [inline app/renderer/css/about.css:52]: Can you add a `cursor: pointer` property here?

File: app/renderer/about.html [hunk 2]
```diff
@@ -7,25 +7,37 @@
...
 	<div class="about">
 		<img class="logo" src="../resources/zulip.png" />
-		<p class="detail" id="version">version ?.?.?</p>
+		<p class="detail" id="version">v?.?.?</p>
 		<div class="maintenance-info">
-			<p class="detail maintainer">Maintained by Zulip</p>
-			<p class="detail license">Available under the Apache License</p>
-			<a class="bug" onclick="linkInBrowser()" href="#">Found bug?</a>
+			<p class="detail maintainer">
+				Maintained by <a onclick="linkInBrowser('repo')">Zulip</a>
```
Comment (akashnimare) [inline app/renderer/about.html]: The URL for Zulip should be https://zulipchat.com, right @rishig ?

**Review by akashnimare — CHANGES_REQUESTED** [review]

**Comment by priyank-p** [comment]
@akashnimare or @rishig can you add GCI Task for this PR Please, Thanks.

**Comment by rishig** [comment]
Thanks for looking into this @cPhost! This already looks much better. 
Before merging: could you post a screenshot of the corresponding pages for slack and nylas?

Also minor comment: Apache License -> Apache 2.0 license.

**Review by akashnimare — APPROVED** [review]

**Comment by akashnimare** [comment]
Slack -

image

**Comment by rishig** [comment]
ok, great. 
Also, I'm not sure this is the most natural place for the "Report an issue" button, but that can be dealt with separately.

**Comment by akashnimare** [comment]
I have put it intentionally there so that users can find it and file the bug. Let's keep it as it is for now. We have the "Report an issue" in `Help` menu item -
 
image

**Comment by priyank-p** [comment]
Just fixed it so it has Apache 2.0 license.

**Comment by akashnimare** [comment]
Merged, thanks @cPhost 🚀

**Pull request merged**